src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
Another good thing is

`validateContext` function will just be

    validateContext(context) {
        return this.getContext(context).valid;
    }